Last-Update: 2015-09-19
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: do not install pre-build .JAR dropped from +DFSG tarball (prevents FTBFS).

--- a/filters/plan/mpxj/planconvert/CMakeLists.txt
+++ b/filters/plan/mpxj/planconvert/CMakeLists.txt
@@ -6,5 +6,5 @@
   "${CMAKE_BINARY_DIR}/filters/plan/mpxj/planconvert/mpxjconfig.h"
 )
 
 
-install(FILES jar/PlanConvert.jar  DESTINATION ${PLANCONVERT_INSTALL_DIR})
+#install(FILES jar/PlanConvert.jar  DESTINATION ${PLANCONVERT_INSTALL_DIR})
